packages/mui-joy/src/ToggleButtonGroup/ToggleButtonGroup.tsx

```diff
@@ -135,43 +136,50 @@ const ToggleButtonGroup = React.forwardRef(function ToggleButtonGroup<
     [value, onChange],
   );
 
+  const toggleButtonGroupContext = React.useMemo(
+    () => ({
+      onClick: (
+        event: React.MouseEvent<HTMLButtonElement>,
+        childValue: string | readonly string[] | number | undefined,
```

Comment:
```suggestion
        childValue: React.ButtonHTMLAttributes<HTMLButtonElement>['value'],
```

Reply:
Done implemented in this commit https://github.com/mui/material-ui/pull/39571/commits/fcd020841eb9ac5e7794309a51a0239f349eeedf